docs/api/settings.md

```diff
@@ -0,0 +1,283 @@
+REST API: Settings
+==================
+
+The settings REST API allows you to view and update WooCommerce Stripe settinngs.
+The endpoints, requests, and responses have similar structure with [WooCommerce core setting options REST API](https://woocommerce.github.io/woocommerce-rest-api-docs/?shell#setting-options).
+
+## Setting option properties
+
+| Attribute     | Type   | Description                           |             |
```

Comment:
Updated in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/314/commits/ef0dabd19bb3b9c764763ec6dc4a3252b33555de.